tools/profiles/develop.json

```diff
@@ -46,7 +46,7 @@
     "IAR": {
         "common": [
             "--no_wrap_diagnostics", "-e",
-            "--diag_suppress=Pa050,Pa084,Pa093,Pa082", "-Oh"],
+            "--diag_suppress=Pa050,Pa084,Pa093,Pa082", "-Oh", "--enable_restrict"],
```

Comment:
Why are we enabling restrict keyword ? The commit does not clarifies this, and cant find any other reference.

This should be already included in C99.

Reply:
Not by default for IAR and CMSIS is using it.